osmosis-1/osmosis.zone_assets.json

```diff
@@ -2493,6 +2493,11 @@
       "chain_name": "osmosis",
       "base_denom": "factory/osmo1pfyxruwvtwk00y8z06dh2lqjdj82ldvy74wzm3/WOSMO",
       "osmosis_verified": false
+    },
+      {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo1279xudevmf5cw83vkhglct7jededp86k90k2le/RAPTR",
+      "osmosis_verified": false
```

Comment:
Please add "osmosis_unlisted": true 
so it can be validated first.
Please move to bottom of the list.